### PR TITLE
[ZH] fixes in Help Center

### PR DESCRIPTION
**Pull request by user670**
### Description

Minor fixes.

### Status

Finished.

### Changelog

- Line 90: fixed the heading level (should be 3, was 2)
- Line 370: removed a quotation mark that shouldn't be there
- Line 847: Fixed link.

### CODE REVIEW

**Comment by benpigchu** [comment]
【 please rename this PR 】

**Comment by kj415j45** [comment]
另外欢迎加入翻译频道: 
https://discord.gg/PjWW8Wg

File: wiki/Help_Center/zh.md [hunk 1]
```diff
@@ -844,7 +844,7 @@ osu! 安装程序自带有 .NET 框架，为了以防万一，您可以[在这
 
 虽然听起来很复杂，但是修复这个问题很简单。
 
-首先，[下载这个压缩包](https://dl.dropboxusercontent.com/u/16332218/goodies.zip)，将它保存在电脑上。
+首先，[下载这个压缩包](https://dl.dropboxusercontent.com/u/16332218/goodies.zip) （链接失效），将它保存在电脑上。
```
Comment (kj415j45) [inline wiki/Help_Center/zh.md]: please apply #580 's change

**Comment by user670** [comment]
Didn't see anything about PR name standards. Changed anyway. 没看到过什么PR命名规范。不过还是改了。

File: wiki/Help_Center/zh.md (hunk 1 above)
Comment (user670) [inline wiki/Help_Center/zh.md]: Don't know how. I'm not that familiar with GitHub, and I'm afraid editing things having the PR applied might mess things up...

Comment (Deardrops) [inline wiki/Help_Center/zh.md]: @user670 把这个链接替换为`https://assets.ppy.sh/store/utilities/osu!go.zip`，commit 后呼叫 @kj415j45 ，之后管理员会帮你merge这个PR。

Comment (user670) [inline wiki/Help_Center/zh.md]: 怎么在这个PR基础上commit？不会弄

**Comment by Deardrops** [comment]
image
点击这个按钮修改文件

**Comment by user670** [comment]
OK.

**Pull request merged**